python/pyarrow/ipc.pxi

```diff
@@ -48,6 +48,26 @@ cdef CMetadataVersion _unwrap_metadata_version(
     raise ValueError("Not a metadata version: " + repr(version))
 
 
+cpdef enum Alignment:
+    Any = <int8_t> CAlignment_Any
```

Comment:
Why are we using `int8_t` here?

Reply:
Right, should be `int64_t` to match C++ Alignment.